src/main/webapp/components/BaseFormView.jsx

```diff
@@ -48,7 +75,9 @@ class BaseFormView extends PureComponent {
         if (props.page === PAGE_INPUT) {
             globalConfig.pages.inputs.services.forEach((service) => {
                 if (service.name === props.serviceName) {
+                    this.groups = service.groups;
                     this.entities = service.entity;
+                    this.groupWiseEntities(service);
```

Comment:
nit: This method is updating `this.entities` but is not clear from its name.

Reply:
Done